website/docs/docs/partnerships.mdx

```diff
@@ -148,23 +148,21 @@ Work with your HashiCorp Alliances representative to verify the plugin within th
 
 ### 5. Support
 
-Getting a new provider built and published to the Terraform Registry is just the first step towards enabling your users with a quality Terraform integration. Once a verified provider has been published, on-going effort is required to maintain the provider.
+Getting a new provider built and published to the Terraform Registry is just the first step towards enabling your users with a quality Terraform integration. Once a partner's provider has been published, on-going effort is required to maintain the provider.
 
-HashiCorp Terraform has an extremely wide community of users and contributors and we encourage everyone to report issues however small, as well as help resolve them when possible. We expect that all verified provider publishers will continue to maintain the provider and address any issues users report in a timely manner. This includes resolving all critical issues within 48 hours and all other issues within 5 business days. HashiCorp reserves the right to remove verified status from any integration that is no longer being maintained.
+HashiCorp Terraform has an extremely wide community of users and contributors and we encourage everyone to report issues however small, as well as help resolve them when possible. We expect that all partner provider publishers will continue to maintain the provider and address any issues users report in a timely manner. This includes resolving all critical issues within 48 hours and all other issues within 5 business days. HashiCorp reserves the right to remove partner status from any integration that is no longer being maintained.
 
-Vendors who choose not to support their provider and prefer to make it a community-supported provider will no longer be listed as Verified.
-
--> **Network Infrastructure Automation (NIA):**  These integrations leverage Consul’s service catalog to seamlessly integrate with Consul-Terraform-Sync (CTS) to automate changes in network infrastructure via a publisher-subscriber method. More details can be found in the [NIA Integration Program](https://www.consul.io/docs/integrate/nia-integration) documentation.
+Vendors who choose not to support their provider and prefer to make it a community-supported provider will no longer be listed as Partner.
```

Comment:
Should they notify us if they want to make this switch?

Reply:
Not really, nobody says I'm losing faith in you and no longer want to support this. There's an issue with deprecation in the industry overall

Reply:
huh also interesting!